test/Common.FunctionalTests/Inprocess/LoggingTests.cs

```diff
@@ -91,5 +91,35 @@ public class LoggingTests : IISFunctionalTestBase
             }
         }
 
+        [ConditionalFact]
+        public async Task StartupMessagesLogFileSwitchedWhenLogFilePresentInWebConfig()
+        {
+            var firstTempFile = Path.GetTempFileName();
+            var secondTempFile = Path.GetTempFileName();
+
+            try
+            {
+                var deploymentParameters = Helpers.GetBaseDeploymentParameters(publish: true);
+                deploymentParameters.EnvironmentVariables["ASPNETCORE_MODULE_DEBUG_FILE"] = firstTempFile;
```

Comment:
How does this work on downlevel IIS?

Reply:
Good point, we will need to skip it if we are win8 or below for full IIS or not run it on IIS.